README.md

```diff
@@ -1,3 +1,7 @@
+<p align="center">
+<img width="250" height="250" src="https://i.imgur.com/jDl9FW8.png">
```

Comment:
Here, we can link to the existing logo on the root of the project 😃

Reply:
Done @AndreVarandas :)

Reply:
Thank you for your time and patience! 🚀